### DISCUSSION TITLE
Using other SVGs in terriajs

### DISCUSSION THREAD

**Discussion by malnajdi** [TerriaJS/terriajs#5447] [Q&A]
despite my tries to override the Icon ReactView to add more SVG icons i was not able to get it to load. Is there special thing i need to do ?

**Comment by malnajdi** [comment]
@steve9164 any help is appreciated

**Comment by zoran995** [comment]
@malnajdi svg icons are included in terriajs source code and then loaded in [Icon.jsx](https://github.com/TerriaJS/terriajs/blob/master/lib/ReactViews/Icon.jsx). 
I don't think there is a way to supply your own icons without rebuilding the TerriaMap. If going that route add icons to `terriajs/wwwroot/images/icons` and then load that image into Icon.[j|t]sx (depending on terriajs version), then you can use it as `Icon.GLYPHS.{icon name}`
